phpcs.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset>
+  <arg value="psv"/>
+  <arg name="extensions" value="php"/>
+
+  <file>src</file>
+  <file>tests</file>
+
+  <exclude-pattern>/vendor/</exclude-pattern>
+
+  <rule ref="PSR12"/>
+
+  <rule ref="PSR2.Classes.PropertyDeclaration.Underscore">
+    <exclude-pattern>/src/</exclude-pattern>
```

Comment:
this is to be excluded in Command.php only
(as it may collide with dynamic args from command line)

in other places $_ is just bad :)

Reply:
Ok